code/modules/materials/Mat_Materials.dm

```diff
@@ -654,30 +680,42 @@ ABSTRACT_TYPE(/datum/material/metal)
 	mat_id = "bohrum"
 	name = "bohrum"
 	desc = "Bohrum is a heavy and highly durable metal."
-	color = "#3D692D"
+	color = list(0.50, 0.00, 0.00, 0.00,\
+				0.25, 0.75, 0.25, 0.00,\
+				0.00, 0.00, 0.50, 0.00,\
+				0.00, 0.00, 0.00, 1.00,\
+				0.00, 0.00, 0.00, 0.00)
 	New()
 		..()
 		setProperty("density", 6)
 		setProperty("hard", 5)
 		setProperty("chemical", 7)
+		addTrigger(TRIGGERS_ON_ADD, new /datum/materialProc/add_color_hsl/bohrum())
+		addTrigger(TRIGGERS_ON_REMOVE, new /datum/materialProc/remove_color_hsl())
 
 
 /datum/material/metal/mauxite
 	mat_id = "mauxite"
 	name = "mauxite"
 	desc = "Mauxite is a sturdy common metal."
-	color = "#534747"
+	color = list(1.00, 0.00, 0.00, 0.00,\
+				0.00, 0.00, 0.00, 0.00,\
+				0.00, 1.00, 1.00, 0.00,\
+				0.00, 0.00, 0.00, 1.00,\
+				0.00, 0.00, 0.00, 0.00)
 	New()
 		..()
 		setProperty("density", 4)
 		setProperty("hard", 3)
+		addTrigger(TRIGGERS_ON_ADD, new /datum/materialProc/add_color_hsl/mauxite())
+		addTrigger(TRIGGERS_ON_REMOVE, new /datum/materialProc/remove_color_hsl())
 
 
 /datum/material/metal/cerenkite
 	mat_id = "cerenkite"
 	name = "cerenkite"
 	desc = "Cerenkite is a highly radioactive metal."
-	color = "#CDBDFF"
+	color = null // "#CDBDFF"
```

Comment:
This is causing a nuclear reactor runtime, it needs to know the colour to apply it to the grid icon